src/dmangle.d

```diff
@@ -88,6 +94,9 @@ private immutable char[TMAX] mangleChar =
     //              X   // variadic T t...)
     //              Y   // variadic T t,...)
     //              Z   // not variadic, end of parameters
+    //              %   // Type backward reference
+    //              ?   // symbol backward reference
+    //              $   // identifier backward reference
```

Comment:
BTW: currently 'Q' is unused, and mangling a TypeIdentifier ('I') doesn't make much sense for me, too. It is supposed to disappear during semantic analysis, so how can it be in a mangled name? Those two letters should be good enough for backward compatible mangling.